app/models/form/lettings/questions/offered.rb

```diff
@@ -7,7 +7,6 @@ def initialize(id, hsh, page)
     @check_answers_card_number = 0
     @max = 150
     @min = 0
-    @hint_text = I18n.t("hints.offered")
```

Comment:
This question is a 2023 question, but the translation for this is missing. We now get an exception because of `config.i18n.raise_on_missing_translations`